src/arche/readers/schema.py

```diff
@@ -42,23 +42,31 @@ def __repr__(self):
 
     def get_enums(self) -> List[str]:
         enums = []
-        for k, v in self.raw["properties"].items():
+        # self.raw["properties"].items() has type:
+        # ItemsView[str, Union[str, bool, int, float, None, list[Any]]]
+        properties = cast(
+            ItemsView[str, Dict[str, Any]], self.raw["properties"].items()
+        )
+        for k, v in properties:
```

Comment:
Why cast here?

Reply:
`self.raw["properties"].items()` has type `ItemsView[str, Union[str, bool, int, float, None, list[Any]]]`, so when we try to get `v.keys()` at line 51, we got error because `str`, `bool`... don't have `.keys()` property

Reply:
As I thought, we can just check if v is `Dict`
```    def get_enums(self) -> List[str]:
        enums: List[str] = []
        for k, v in self.raw["properties"].items():
            if isinstance(v, Dict) and "enum" in v.keys():
                enums.append(k)
        return enums
```